ui/app/css/variables/colors.scss

```diff
@@ -1,5 +1,5 @@
 // These are the colors of the MetaMask design system
-// Only design system colors should be added, no superfulous variables
+// Only design system colors should be added, no superfluous variables
```

Comment:
*checks blame* I find this change to be Superfluous.

Reply:
This should've been in #9038 😅

Reply:
lol, no worries. I'm just mildly concerned that the vscode plugin that prevents me from contributing a wide array of typos has failed.